featuretools/primitives/standard/datetime_transform_primitives.py

```diff
@@ -201,7 +201,7 @@ def distance_to_holiday(x):
             df["x_index"] = df.index  # store original index as a column
             df = df.dropna()
             df = df.sort_values("date")
-            df.date = df.date.dt.normalize()
+            df.date = df.date.dt.normalize().dt.date.astype("datetime64[ns]")
```

Comment:
do we need to normalize if we are also extracting the date?

Reply:
No, the normalize calls aren't necessary anymore. Removed. Also updated to (I think) improve readability of these lines a little bit.